static/app/bootstrap/initializeApp.tsx

```diff
@@ -1,6 +1,4 @@
-import 'bootstrap/js/alert';
```

Comment:
https://github.com/twbs/bootstrap/blob/v3.4.1/js/alert.js

Reply:
This looks more complicated than it is because of all the transition stuff, but it's legit just calling `remove()` on the closest `.alert` element.

The replacment for this is just stuff like `this.parentElement.remove()`.